plugins/woocommerce/src/Blocks/BlockTypes/BlockifiedProductDetails.php

```diff
@@ -13,6 +13,31 @@ class BlockifiedProductDetails extends AbstractBlock {
 	 */
 	protected $block_name = 'blockified-product-details';
 
+	public function add_block_type_metadata( $metadata ) {
+		if ( $metadata['name'] === 'woocommerce/accordion-group' ) {
+			$metadata['attributes']['__isProductDetailsChildren'] = array(
+				'type'    => 'boolean',
+				'default' => false,
+			);
+			return $metadata;
+		}
+		return $metadata;
```

Comment:
Updating the attributes of `woocommerce/accordion-group` to support injecting a custom attribute, enabling filtering of instances that are children of the Product Details block.